.github/workflows/validate_version.yaml

```diff
@@ -0,0 +1,24 @@
+name: Validate Release
+
+on:
+  deployment:
+  pull_request:
+
+
+jobs:
+  validate-Release-version:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Validate Release Version
+        run: .github/scripts/validate_release_version.sh
```

Comment:
this performs as your previous action, i don't under stand this duplications, please explain

Reply:
This runs only if the user opens a PR to check if and  inform the user to update the version number